app/components/Headers/UnderlinedHeader/index.jsx

```diff
@@ -0,0 +1,29 @@
+// @flow
```

Comment:
Although now slightly inconsistent in our code base lets have index.jsx be the file that exports the component and includes any redux logic or HOC and not have it be the file that contains any jsx markup.

So please add a file `UnderlinedHeader.jsx` and then export it from `index.js` 

Reply:
Makes complete sense, I'll change that now and stick to convention in the future 👍